src/components/manage/Display/__snapshots__/Display.test.jsx.snap

```diff
@@ -1,69 +1,48 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Display renders an actions component 1`] = `
-Array [
+<div>
   <label
-    htmlFor="display-select"
```

Comment:
@sneridagh can you explain why the snapshots contain the HTML version (for, class) instead of the React version (htmlFor, className)?

Reply:
I used @testing-library on them instead of the react-testing-renderer, and it outputs proper html.

I think we should move all of them to use it, sooner or later, and force all new developments to use it.

Reply:
Ok. That makes a lot of sense. In the future, it might be better to create a separate PR for this move to make it obvious where this comes from. Not worth the effort for this PR now I guess though.